js/bytex.js

```diff
@@ -670,60 +793,56 @@ module.exports = class bytex extends Exchange {
         ];
     }
 
+    parseBalance (response) {
+        const timestamp = this.parse8601 (this.safeString (response, 'updated_at'));
+        const result = {
+            'info': response,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+        };
+        const currencyIds = Object.keys (this.currencies_by_id);
+        for (let i = 0; i < currencyIds.length; i++) {
+            const currencyId = currencyIds[i];
+            const code = this.safeCurrencyCode (currencyId);
+            const account = this.account ();
+            account['free'] = this.safeString (response, currencyId + '_available');
+            account['total'] = this.safeString (response, currencyId + '_balance');
+            result[code] = account;
+        }
+        return this.safeBalance (result);
+    }
+
```

Comment:
Try to keep things in the same place in the future, reviewing this becomes more difficult when things get moved around because I don't know what has changed, and I have to scroll up and down to compare them. In the picture below you can see that the red is deleted code, and the green is added code, but they're actually the same code, so nothing has changed, but I have to compare the methods line by line to make sure that's the case

![](https://i.imgur.com/29ArmHd.png)

Reply:
My apologies, this will not happen again.

Reply:
> My apologies, this will not happen again.

don't worry about it